tools/install.py

```diff
@@ -160,6 +160,10 @@ def headers(action):
     'src/node_version.h',
   ], 'include/node/')
 
+  # Add the expfile that is created on AIX
+  if 'aix' in sys.platform:
```

Comment:
sorry again, but may be `startswith` like [this](https://github.com/mhdawson/io.js/blob/aix2/configure#L558) is better than 'in'? currently no side effects but may be not so future safe?